Main.java

```diff
@@ -0,0 +1,73 @@
+package com.company;
+
+import java.io.*;
+import java.util.StringTokenizer;
+
+public class Main {
+
+    public static void main(String[] args){
+
+        if(args.length!=2)
+        {
+            System.out.println("File path and search word cannot be empty");
+            System.exit(0);
+        }
+        String path=args[0];
+        String search=args[1];
+        File file=new File(path);
+        System.out.println("Processing...");
+        doesFileExist(file);

```

Comment:
By this logic, the wordCount method will be called even if the file does not exist. You should either throw an exception from doesFileExist or 
Make it return a boolean, check the return value and only then call wordCount

Reply:
I have made doesFileExist() return a boolean value. wordCount() will be called only if doesFileExist() true.